testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -1553,6 +1578,12 @@ fn multiple_miners() {
         u64::try_from(miner_1_tenures + miner_2_tenures).unwrap()
     );
 
+    rl2_coord_channels
```

Comment:
I was not 100% sure this was necessary either, but we were not properly terminating the runloop for the second node so I added it anyway.